docs/rackhd/configuration.rst

```diff
@@ -170,6 +170,268 @@ configuration. Custom certificates can also be used with some configuration.
+that was added to the **Sku Discovery graph** in the first step.
+Below is a snippet of the Discovery graph without the reboot task (the reboot task was originally located
+after the task 'catalog-lldp')
+
+.. code-block:: javascript
+
+   module.exports = {
+    friendlyName: 'Discovery',
+    injectableName: 'Graph.Discovery',
+    options: {
+        'bootstrap-ubuntu': {
+            'triggerGroup': 'bootstrap'
+        },
+        'finish-bootstrap-trigger': {
+            'triggerGroup': 'bootstrap'
+        }
+    },
+    tasks: [
+        {
+            label: 'bootstrap-ubuntu',
+            taskName: 'Task.Linux.Bootstrap.Ubuntu'
+        },
+        {
+            label: 'catalog-dmi',
+            taskName: 'Task.Catalog.dmi'
+        },
+        {
+            label: 'catalog-ohai',
+            taskName: 'Task.Catalog.ohai',
+            waitOn: {
+                'catalog-dmi': 'finished'
+            }
+        },
+        {
+            label: 'catalog-bmc',
+            taskName: 'Task.Catalog.bmc',
+            waitOn: {
+                'catalog-ohai': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-lsall',
+            taskName: 'Task.Catalog.lsall',
+            waitOn: {
+                'catalog-bmc': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-megaraid',
+            taskName: 'Task.Catalog.megaraid',
+            waitOn: {
+                'catalog-lsall': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-smart',
+            taskName: 'Task.Catalog.smart',
+            waitOn: {
+                'catalog-megaraid': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-driveid',
+            taskName: 'Task.Catalog.Drive.Id',
+            waitOn: {
+                'catalog-smart': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-lldp',
+            taskName: 'Task.Catalog.LLDP',
+            waitOn: {
+                'catalog-driveid': 'finished'
+            },
+            ignoreFailure: true
+        },
+       {
+            label: 'finish-bootstrap-trigger',
+            taskName: 'Task.Trigger.Send.Finish',
+            waitOn: {
+                'catalog-lldp': 'finished'
+            }
+        }
+    ]
+   };
+
+
+Once the above steps are completed (edited and saved) the services need to be restarted:
+
+.. code-block:: shell
+
+    sudo service on-http start
+    sudo service on-dhcp-proxy start
+    sudo service on-syslog start
+    sudo service on-taskgraph start
+    sudo service on-tftp start
+
+Once the services are restarted completely, running an ipmi command for the user list should show the new user added.
```

Comment:
Minor, but in the spirit of not being too pedantic, is this part necessary?  ...since RackHD abstracts all this for the user.